src/multi/multi_out.rs

```diff
@@ -30,16 +30,30 @@ impl Output for MultiOutput {
 }
 
 impl MultiOutput {
-    /// Create a new multi-output.
-    pub fn output() -> MultiOutput {
+    /// Create a new multi-output dispatcher.
+    #[deprecated(since="0.7.2", note="Use new()")]
+    pub fn output() -> Self {
+        Self::new()
+    }
+
+    /// Create a new multi-output dispatcher.
+    pub fn new() -> Self {
         MultiOutput {
             attributes: Attributes::default(),
             outputs: vec![],
         }
     }
 
-    /// Returns a clone of the dispatch with the new output added to the list.
+    /// Add a target to the dispatch list.
+    /// Returns a clone of the original object.
+    #[deprecated(since="0.7.2", note="Use target()")]
     pub fn add_target<OUT: Output + Send + Sync + 'static>(&self, out: OUT) -> Self {
+        self.target(out)
+    }
+
+    /// Add a target to the dispatch list.
+    /// Returns a clone of the original object.
+    pub fn target<OUT: Output + Send + Sync + 'static>(&self, out: OUT) -> Self {
```

Comment:
I wondered about this too, but went all the way for consistency with the other changes. I think sticking with `add_target` would also imply using `add_name` instead of `named` on `Attributes`, because both method essentially act the same way, cloning & appending the element to an internal list.